src/Commands/DataExplorer/ClusterListCommand.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Arguments.DataExplorer;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+using System.CommandLine.Parsing;
+
+namespace AzureMcp.Commands.DataExplorer;
+
+public sealed class ClusterListCommand : SubscriptionCommand<ClusterListArguments>
+{
+    private readonly ILogger<ClusterListCommand> _logger;
+
+    public ClusterListCommand(ILogger<ClusterListCommand> logger) : base()
+    {
+        _logger = logger;
+    }
+
+    protected override string GetCommandName() => "list";
+
+    protected override string GetCommandDescription() =>
+        """
+        List all Data Explorer clusters in a subscription. This command retrieves all clusters 
+        available in the specified subscription. Results include cluster names and are returned as a JSON array.
```

Comment:
```suggestion
        available in the specified subscription. Result is a list of cluster names as a JSON array.
```

remove the ambiguity of "includes"